context/app/api/vitessce_confs/test_confs.py

```diff
@@ -0,0 +1,49 @@
+import json
+import glob
+from pathlib import Path
+import re
+
+from .base_confs import ImagePyramidViewConf
+from .assay_confs import (
+    SeqFISHViewConf,
+    CytokitSPRMConf,
+    RNASeqConf,
+    ATACSeqConf,
+    IMSConf,
+)
+
+MOCK_NEXUS_TOKEN = "nexus_token"
+
+FIXTURES_INPUT_DIR = "fixtures/input_entity"
+FIXTURES_EXPECTED_OUTPUT_DIR = "fixtures/output_expected"
+
+AssayConfClasses = {
+    "codex": CytokitSPRMConf,
+    "rna": RNASeqConf,
+    "atac": ATACSeqConf,
+    "ims": IMSConf,
+    "image_pyramid": ImagePyramidViewConf,
+    "seqfish": SeqFISHViewConf,
+}
+
+
+def test_assays():
+    parent_dir = Path(__file__).parent
+    for entity_file in glob.glob(
+        str(parent_dir / f"{FIXTURES_INPUT_DIR}/*_entity.json")
+    ):
```

Comment:
Assuming `FIXTURES_INPUT_DIR` doesn't contain `*`s:
```suggestion
    for entity_file in (parent_dir / FIXTURES_INPUT_DIR).glob('*_entity.json'):
```

Reply:
When doing it this way, I like to have a check that there is at least one fixture... otherwise if the path is wrong, and there are no matches, there won't be any failure... but it's not running the tests is should, either.